DEVELOPMENT.md

```diff
@@ -381,12 +381,69 @@ kubectl set env deployments --all SYSTEM_NAMESPACE=${TARGET_NAMESPACE} -n ${TARG
 ```
 
 This script will cause `ko` to:
+
 - Change (resolve) all `namespace` values in K8s configuration files within the `config/` subdirectory to be updated to a name of your choosing.
 - Builds and push images with the new namespace to your container registry and
 - Update all Tekton Objects in K8s using these images
 
 It will also update the default system namespace used for K8s `deployments` to the new value for all subsequent `kubectl` commands.
 
+---
+
+### Standing up a K8s cluster with Tekton using the `kind` tool
+
+An alternative to standing up your own K8s cluster and installing Tekton using `ko` is by using the [kind](https://kind.sigs.k8s.io/) tool. It was designed to help create and run local,  Kubernetes clusters in Docker to assist in local development and testing.
+
+The [Tekton "plumbing" project](https://github.com/tektoncd/plumbing) provides a convenience script, named ['tekton_in_kind.sh'](https://github.com/tektoncd/plumbing/blob/main/hack/tekton_in_kind.sh), that leverages `kind` to create a cluster and then deploy Tekton Pipeline, [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components into it.
+
+See Tekton Plumbing's [DEVELOPMENT.md](https://github.com/tektoncd/plumbing/blob/main/DEVELOPMENT.md) for more details on this and other helpful scripts and tools.
+
+#### Installation and prerequisites
+
+- Clone the [Tekton Plumbing](https://github.com/tektoncd/plumbing) repository which has the ['tekton_in_kind.sh'](https://github.com/tektoncd/plumbing/blob/main/hack/tekton_in_kind.sh) and other helpful scripts.
+- `kind`: Install using its ["quick start"](https://kind.sigs.k8s.io/docs/user/quick-start/) documentation.
+- `Docker`: `kind` also requires Docker to be installed and running locally. Use the [Get Docker](https://docs.docker.com/get-docker/) instructions.
+
+#### Create a cluster with Tekton components
+
+Change into the Tekton plumbing repository you cloned and invoke the script:
+
+```shell
+cd plumbing
+./hack/tekton_in_kind.sh
+```
```

Comment:
I haven't tried these instructions myself but excited to experiment with it. Thank you @mrutkows for documenting them in the pipeline repo.